package Exception;

public class Arithmatic_Exception {
	public static void main(String[] args) {
		        int a=1;
				int b=0;
			
						
				try {//risky code inside the try block
					
				int	c=a/b;
				System.out.println(c);
					
				}

	catch (Throwable monkey){
		
		System.out.println("Exception is hndeled");
	}
				System.out.println("Lokesh");
				System.out.println("hiii");
		
	}
	}
	


